src/components/Book.jsx

```diff
@@ -11,17 +11,53 @@ const Book = ({ item }) => {
     dispatch(removeBook(item.item_id));
   };
   return (
-    <div>
-      <h3>{item.title}</h3>
-      <p>
-        Author:
-        {' '}
-        {item.author}
-      </p>
-      <button type="button" onClick={handleDelete}>
-        Delete
-      </button>
-    </div>
+    <section className="contBookSurrounding">
+      <div className="contBook">
+        <span className="category">Action</span>
+        <h3 className="Title">{item.title}</h3>
+        <span className="Suzanne-Collins">
+          Author:
+          {' '}
+          {item.author}
+        </span>
+        <div className="allButtons">
+          <button className="CommentsButton" type="button">Comments</button>
+          <button className="RemoveButton" type="button" onClick={handleDelete}>
+            Remove
+          </button>
+          <button className="EditButton" type="button">Edit</button>
+        </div>
+      </div>
+      <div className="Advance">
+        <div className="contCircularBar">
+          <div className="circle-wrap">
+            <div className="circle">
+              <div className="mask half">
+                <div className="fill">
+                  <div className="mask full">
+                    <div className="inside-circle" />
+                  </div>
+                </div>
+              </div>
+            </div>
```

Comment:
- [ ] It looks good that you added a circular progress bar, but it is smaller compared to the design. Please consider updating it.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/eamador1/bookstore-redux/assets/55840999/4787d3d1-2c23-4002-a1a6-cbd521168b9a)| ![image](https://github.com/eamador1/bookstore-redux/assets/55840999/b60d0723-ec24-4302-83ad-b7dc6583237a)|